src/web/layouts/ImmersiveLayout.tsx

```diff
@@ -273,12 +272,12 @@ export const ImmersiveLayout = ({
                         100vw is normally enough but don't let the content shrink vertically too
                         much just in case
                     */
-					min-height: 25rem;
+					min-height: ${mainMedia && '25rem'};
```

Comment:
if `mainMedia` is not defined does this set `min-height: undefined`? if so maybe worth not setting the key?

```suggestion
					${mainMedia && 'min-height: 25rem'};
```

Reply:
Looks like doing this throws an error inside the `css` template. It needs a colon after a string to register as correct

Reply:
Opting for
```javascript
const noMainMediaStyles = css`
	height: 100vh;
	/**
    100vw is normally enough but don't let the content shrink vertically too
    much just in case
    */
	min-height: 25rem;
	${from.desktop} {
		min-height: 31.25rem;
	}
	${from.wide} {
		min-height: 50rem;
	}
`;
```
and then using 
```javascript
className={cx(
    !mainMedia && noMainMediaStyles,
    css`
        display: flex;
	flex-direction: column;
    `,
)}
```

Reply:
yeah that looks a lot nicer! 👍